src/txn/txn_rollback_to_stable.c

```diff
@@ -126,6 +130,62 @@ __txn_abort_newer_insert(
         __txn_abort_newer_update(session, ins->upd, rollback_timestamp);
 }
 
+/*
+ * __txn_abort_fix_row_on_disk_kv --
+ *     Fix the on-disk row K/V version according to the given timestamp.
+ */
+static int
+__txn_abort_fix_row_on_disk_kv(
+  WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW *rip, wt_timestamp_t rollback_timestamp)
```

Comment:
I would avoid the use of `fix` in the function name here because it usually implies `fixed-length column store` everywhere else in WiredTiger. I suggest just removing that and using `txn_abort_row_ondisk_kv`.

Reply:
Changed the function name as per the suggestion.